lib/absinthe/schema/notation/sdl_render.ex

```diff
@@ -5,13 +5,11 @@ defmodule Absinthe.Schema.Notation.SDL.Render do
 
   alias Absinthe.Blueprint
 
-  @line_width 120
-
-  def inspect(term, %{pretty: true}) do
+  def inspect(term, %{pretty: true, width: width}) do
```

Comment:
This will only match if a width is passed as an opt right?  Do you also need another clause that does not match on width?

Reply:
I believe we benefit here from the default [Inspect.Opts](https://hexdocs.pm/elixir/Inspect.Opts.html).